Cargo.toml

```diff
@@ -33,9 +33,9 @@ as_variant = "1.2.0"
 base64 = "0.22.1"
 byteorder = "1.5.0"
 chrono = "0.4.39"
-eyeball = { version = "0.8.8", features = ["tracing"] }
-eyeball-im = { version = "0.6.0", features = ["tracing"] }
-eyeball-im-util = "0.8.0"
+eyeball = { git = "https://github.com/Hywan/eyeball", branch = "feat-im-util-skip", features = ["tracing"] }
+eyeball-im = { git = "https://github.com/Hywan/eyeball", branch = "feat-im-util-skip", features = ["tracing"] }
+eyeball-im-util = { git = "https://github.com/Hywan/eyeball", branch = "feat-im-util-skip" }
```

Comment:
We should probably work on these patches from a matrix-org organisation fork as opposed to your personal one.

Reply:
Yes! [My patch on eyeball has been merged](https://github.com/jplatte/eyeball/pull/72/), and I'm about to make a release of `eyeball-im-util`, which would remove the need for this change in our `Cargo.toml` file.